README.md

```diff
@@ -153,6 +153,61 @@ The purpose of synapseutils is to create a space filled with convenience functio
         print(dirname)
         print(filename)
 
+OpenTelemetry (OTEL)
+--------------------------------
+[OpenTelemetry](https://opentelemetry.io/) is a tool that can be used to collect telemetry data. The synapseclient is ready to provide traces should you want them. The Synapse Python client supports OTLP Exports and can be configured via environment variables as defined [here](https://opentelemetry-python.readthedocs.io/en/stable/exporter/otlp/otlp.html).
```

Comment:
I think we can provide a little purpose around collecting telemetry data. Something like: "Analysis of traces and spans can provide insights into latency, errors, and other performance metrics." (pulled directly from google)

Reply:
+1. I am swapping this paragraph to read as:
```suggestion
[OpenTelemetry](https://opentelemetry.io/) helps support the analysis of traces and spans which can provide insights into latency, errors, and other performance metrics. The synapseclient is ready to provide traces should you want them. The Synapse Python client supports OTLP Exports and can be configured via environment variables as defined [here](https://opentelemetry-python.readthedocs.io/en/stable/exporter/otlp/otlp.html).
```